src/FindTranslocations.cpp

```diff
@@ -15,9 +16,127 @@
 
 #include "data_structures/Translocation.h"
 #include  <boost/program_options.hpp>
+#include "api/BamWriter.h"
+
 namespace po = boost::program_options;
 
 
+//returns a bam file contaning the reads located in a given region of the genome
+void TranslocationBAM(string BamFileName,string outputFileHeader,string positionData){
+	int chr[2];
+	int startPos[2];int endPos[2];
+	int arguments;
+
+	//processing the command line input
+	string buf; 
+	stringstream ss(positionData);
+	vector<string> tokens;
+	while (ss >> buf){tokens.push_back(buf);}
+	arguments=tokens.size();
+
+
+	string tmpstr;
+	if(arguments == 6){
+		try{
+			for(int i=0;i<=1;i++){
```

Comment:
This is strange, the for loop seems not needed as you iterate it at most once being the condition i<=1